dace/sdfg/nodes.py

```diff
@@ -217,6 +217,10 @@ class AccessNode(Node):
     debuginfo = DebugInfoProperty()
     data = DataProperty(desc="Data (array, stream, scalar) to access")
 
+    instrument = EnumProperty(dtype=dtypes.DataInstrumentationType,
```

Comment:
Goes into the same as the previous comments on this: the `instrument` property is now of two different types depending on what node it belongs to. If (and only if) the two were to be merged, it would maybe make sense to make `instrument` a general property for all Node (or even element) types.